### PR TITLE
Added an implementation in Smalltalk

### PR DESCRIPTION
**Pull request by fvbakel**
## Description
Added an implementation in Smalltalk. See README.md

## Contributing requirements
* [X] I read the contribution guidelines in CONTRIBUTING.md.
* [X] I placed my solution in the correct solution folder.
* [X] I added a README.md with the right badge(s).
* [X] I added a Dockerfile that builds and runs my solution.
* [X] I selected `drag-race` as the target branch.
* [X] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by marghidanu** [comment]
Thank you for another contribution! The only few things I saw are just formatting issues, I'm sure you can run a code formatter on the source code otherwise it looks good to me.

File: PrimeSmalltalk/solution_1/primes.st [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+start := Time millisecondClockValue.
+
+duration_ms := 0.
+passes := 0.
+[ duration_ms < max_limit_ms ] whileTrue: [
+    passes := passes +1.
+    sieve := PrimeSieve new: limit.
+    sieve run_sieve.
+
+    duration_ms := (Time millisecondClockValue) -start.
```
Comment (marghidanu) [inline PrimeSmalltalk/solution_1/primes.st]: Maybe a space after the minus sign.

File: PrimeSmalltalk/solution_1/primes.st [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+    (duration_ms >= max_limit_ms)
+        ifTrue: [
+            duration := duration_ms / 1000.
+            sieve print_results: show_results time: duration nr: passes.
+        ].
+].
+
```
Comment (marghidanu) [inline PrimeSmalltalk/solution_1/primes.st]: Just one new line should be fine, right? :)

File: PrimeSmalltalk/solution_1/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:18.04
+
+# small talk is not on Alpine repositories
```
Comment (rbergen) [inline PrimeSmalltalk/solution_1/Dockerfile]: I believe this is not a comment on apt-get. but on the selection of ubuntu as the base image. If I'm right, I would make this comment the first line of the Dockerfile, or remove it altogether (I don't think a code-level "justification" of this decision is necessary, tbh.)

File: PrimeSmalltalk/solution_1/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+RUN apt-get update -qq \
+    && apt-get install -y gnu-smalltalk
+
+WORKDIR /opt/app
+
+COPY ./primes.st .
+
+ENTRYPOINT ["gst","./primes.st"]
+#ENTRYPOINT ["/bin/bash"]
```
Comment (rbergen) [inline PrimeSmalltalk/solution_1/Dockerfile]: I think it's cleaner to remove the (default?) /bin/bash ENTRYPOINT than it is to comment it out.

File: PrimeSmalltalk/solution_1/primes.st (hunk 1 above)
Comment (fvbakel) [reply PrimeSmalltalk/solution_1/primes.st]: Ah, solved spaces and new lines in various places.

File: PrimeSmalltalk/solution_1/primes.st (hunk 2 above)
Comment (fvbakel) [reply PrimeSmalltalk/solution_1/primes.st]: Indeed.

File: PrimeSmalltalk/solution_1/Dockerfile (hunk 3 above)
Comment (fvbakel) [reply PrimeSmalltalk/solution_1/Dockerfile]: Agree, removed the comment.

File: PrimeSmalltalk/solution_1/Dockerfile (hunk 4 above)
Comment (fvbakel) [reply PrimeSmalltalk/solution_1/Dockerfile]: Yes, I agree. I used that for debugging and troubleshooting.

**Comment by fvbakel** [comment]
Thank you for both your quick reviews. I applied the fixes.

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**